x-pack/solutions/security/plugins/security_solution/public/attack_discovery/pages/index.tsx

```diff
@@ -245,6 +251,10 @@ const AttackDiscoveryPageComponent: React.FC = () => {
 
         <EuiSpacer size="s" />
 
+        {attacksAlertsAlignmentEnabled && <MovingAttacksCallout />}
+
+        <EuiSpacer size="s" />
```

Comment:
Consider rendering the `EuiSpacer` only when the FF is enabled, because when compared to `main`, this appears to add extra space when the FF is disabled:

<img width="3071" height="460" alt="main_vs_pr" src="https://github.com/user-attachments/assets/92d35988-8e0f-430c-9bd2-7a9033a8a439" />

Reply:
Done in https://github.com/elastic/kibana/pull/243272/commits/2a4874dd8f98ef13269b893bc7fbdacb63a10b53